spec.md

```diff
@@ -3,193 +3,138 @@ title: "HTTP API V2"
 description: "Specification for the Registry API."
 keywords: registry, on-prem, images, tags, repository, distribution, api, advanced
 ---
-
-# Docker Registry HTTP API V2
-
-## Introduction
-
-The _Docker Registry HTTP API_ is the protocol to facilitate distribution of images to the docker engine.
-It interacts with instances of the docker registry, which is a service to manage information about docker images and enable their distribution.
-The specification covers the operation of version 2 of this API, known as _Docker Registry HTTP API V2_.
-
-While the V1 registry protocol is usable, there are several problems with the architecture that have led to this new version.
-The main driver of this specification is a set of changes to the Docker image format, covered in [docker/docker#8093](https://github.com/docker/docker/issues/8093).
-The new, self-contained image manifest simplifies image definition and improves security.
-This specification will build on that work, leveraging new properties of the manifest format to improve performance, reduce bandwidth usage and decrease the likelihood of backend corruption.
-
-For relevant details and history leading up to this specification, please see the following issues:
+# Open Container Initiative
+## Distribution Specification
+
+This specification defines an API protocol to facilitate distribution of images.
+
+The goal of this specification is to standardize container image distribution based on the specification for the [Docker Registry HTTP API V2 protocol](https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md).
```

Comment:
Would it not be useful to have pointers to the previous docker-registry v1 as historical defacto info?

Reply:
I don't think we should mention that in the intro. The protocol inherited very little from v1 registry. If we have a section about the background it could be included, as well as the layer design and move to content addressability.

Reply:
yeah fair.